src/test/java/uk/gov/companieshouse/overseasentitiesapi/validation/OverseasEntitySubmissionDtoValidatorTest.java

```diff
@@ -588,70 +618,132 @@ void testPartialUpdateValidationNoFilingDate() {
     void testFullUpdateValidationWithoutTrusts() {
         buildOverseasEntitySubmissionDto();
 
-        testFullUpdateRemoveValidationWithoutTrusts();
+        Errors errors = testFullUpdateRemoveValidationWithoutTrusts();
+        assertFalse(errors.hasErrors());
     }
 
     @Test
-    void testFullRemoveValidationWithoutTrusts() {
+    void testFullRemoveValidationWithoutTrustsFilingDateNotPresent() {
         buildOverseasEntitySubmissionDto();
         overseasEntitySubmissionDto.setIsRemove(true);
+        overseasEntitySubmissionDto.getUpdate().setFilingDate(null);
+        Errors errors = testFullUpdateRemoveValidationWithoutTrusts();
 
-        testFullUpdateRemoveValidationWithoutTrusts();
+        assertFalse(errors.hasErrors());
+        verify(removeValidator, times(1)).validateFull(any(), any(), any());
+    }
 
+    @Test
+    void testFullRemoveValidationWithoutTrustsFilingDateIsPresent() {
+        buildOverseasEntitySubmissionDto();
+        overseasEntitySubmissionDto.setIsRemove(true);
+        Errors errors = testFullUpdateRemoveValidationWithoutTrusts();
+
```

Comment:
on 640 the errors returned don't get checked for no errors (it used to be checked in testFullUpdateRemoveValidationWithoutTrusts). I would keep the asserts in the methods where you return the Errors, that way it will assert and also return the Errors as well.

Reply:
We have an assert error check where we expect errors - which we do for all remove scenarios where a filing date (by default) is present (see below)

Reply:
I wrongly thought that the Errors object returned should be free of errors but in this case it should contain the error that is checked in the lines after this one.